client/src/components/FriendBox.js

```diff
@@ -47,8 +47,9 @@ const FriendBox = () => {
 
     useEffect(() => {
         if(data) {
-            setFriendList([...data.getFriends[0].friends]);
```

Comment:
I'm wondering why this is commented out, this is what makes the friends list work